### ISSUE TITLE
Where is the upstream issue tracker?

### ISSUE THREAD

**Issue by roke-julian-lockhart** [Distrotech/iw#2]
This is a mirror, so issues here are really about the mirror, I'm guessing.

#### Context

I ask because:

```sh
#!/usr/bin/env sh
sudo iw phy0 wowlan show
```

...causes:

> ```YAML
> command failed: Operation not supported (-95)
> ```

...to appear. This is really weird, because these devices support WOL in their BIOSes/(U)EFIs (unsure which they use).

The sole indexed places on the internet where the command appears appears to be

1. [`forum.manjaro.org/t/123329`](https://forum.manjaro.org/t/iw-dev-connect-operation-not-supported-95/123329?u=beedellrokejulianloc#:~:text=key%200%3A0544880620-,command%20failed%3A%20operation%20not%20supported%20(-95),-%24%20sudo%20iw)

1. [`cyberciti.biz/faq/configure-wireless-wake-on-lan-for-linux-wifi-wowlan-card`](https://www.cyberciti.biz/faq/configure-wireless-wake-on-lan-for-linux-wifi-wowlan-card/#respond:~:text=sudo%20iw%20phy0%20wowlan%20show)

1. [`linuxquestions.org/questions/linux-networking-3/ibss-join-returns-operation-not-supported-95-a-4175649573/#post5970439`](https://www.linuxquestions.org/questions/linux-networking-3/ibss-join-returns-operation-not-supported-95-a-4175649573/#post5970439:~:text=command%20failed%3A%20operation%20not%20supported%20(-95))

**Comment by maxence-eero** [comment]
Ever figure this out? IW segfaults for a certain command and while I fixed it it seems weird that it was never patched

**Comment by beedell-roke** [comment]
> https://github.com/Distrotech/iw/issues/2#issuecomment-2941830794

@maxence-eero, no. I suppose that we can utilise as an impromptu tracker, since you can always mark a comment as off-topic. I'd say post the `bt all` (and coredump, if < 21 MiB) here, and we'll locate the tracker later.

**Comment by beedell-roke** [comment]
> https://github.com/Distrotech/iw/issues/2#issuecomment-2941867552

Actually, @maxence-eero, after some very nominal investigation, I've ascertained that `iw` is maintained by the kernel developers, so the tracker should be [`bugzilla.kernel.org/enter_bug.cgi?product=Networking&component=Wireless`](https://bugzilla.kernel.org/enter_bug.cgi?product=Networking&component=Wireless). I see bugs involving `iw` there.
